components/microfrontend/hack/generate-groups.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
It is not needed as you use `./vendor/k8s.io/code-generator/generate-groups.sh`

Reply:
done